lld/ELF/Config.h

```diff
@@ -476,6 +476,12 @@ struct Ctx {
   // True if we need to reserve two .got entries for local-dynamic TLS model.
   std::atomic<bool> needsTlsLd{false};
 
+  // Each symbol assignment and reference is assigned an increasing order.
```

Comment:
By reference, do you mean the Defined(sym) reference. I couldn't see any other place that a symbol reference would get an increasing order.

If I'm right perhaps // Each symbol assignment and DEFINED(symbol) reference is assigned an increasing order.